backend/taxHandlers.ts

```diff
@@ -18,7 +18,7 @@ const getDiscountForLine = (
   totalDiscount: number,
   allLinesTotal: number
 ) => {
-  if (totalDiscount === 0 || allLinesTotal == 0) {
+  if (totalDiscount === 0 || allLinesTotal === 0) {
```

Comment:
💡 if you want to avoid issues with `==` instead of `===` you could add this rule in ESLint https://eslint.org/docs/latest/rules/eqeqeq ;)

Reply:
Added